test/test-form.js

```diff
@@ -286,6 +286,58 @@ test('handle empty object', function (t) {
     t.end();
 });
 
+test('promised handle empty', function (t) {
+    // eslint-disable-next-line no-undef
+    if (typeof Promise === 'function') {
+        return t.end();
+    }
```

Comment:
```suggestion
test('promised handle empty', { skip: typeof Promise !== 'function' }, function (t) {
```
for the eslint warning, let's add it as a global to eslintrc